rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -851,20 +851,20 @@ def samples(self):
 
         return self._samples
 
-    def wait(self, timeout=None, return_loan=True):
+    def wait(self, timeout=None, return_samples=False):
         """Wait for this input to receive data.
 
         This method waits for the specified timeout for data to be received by
         this input.
         If the operation times out, it raises :class:`TimeoutError`.
 
         :param number timeout: The maximum time to wait in milliseconds. By default, infinite.
-        :param bool return_loan: Whether to return outstanding loans before waiting. By default, ``True``.
+        :param bool return_samples: Whether to return outstanding loans before waiting. By default, ``False``.
```

Comment:
Whether to clear the data in Input.samples before waiting. By default it's ``False`` for backward compatibility. It is recommended to set it to ``True``.